internalshared/configutil/telemetry_test.go

```diff
@@ -0,0 +1,19 @@
+package configutil
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestParsePrefixFilters(t *testing.T) {
+	prefixFilters := []string{"", "+vault.abc", "-vault.abc", "vault.abc"}
+
+	allowedPrefixes, blockedPrefixes := parsePrefixFilter(prefixFilters)
+
+	assert.Equal(t, len(allowedPrefixes), 1)
```

Comment:
You can say `assert.Len` to check the length with testify.

Reply:
I ended up just doing an `assert.Equal` against an expected array, while refactoring the test
6a28af2